src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/PrisonerOffenderSearch.kt

```diff
@@ -1,11 +1,16 @@
 package uk.gov.justice.digital.hmpps.prisonersearch
 
 import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.autoconfigure.data.elasticsearch.ReactiveElasticsearchRepositoriesAutoConfiguration
+import org.springframework.boot.autoconfigure.data.elasticsearch.ReactiveElasticsearchRestClientAutoConfiguration
 import org.springframework.boot.runApplication
 
-@SpringBootApplication
+@SpringBootApplication(
+  exclude = [ReactiveElasticsearchRestClientAutoConfiguration::class,
```

Comment:
have to exclude these otherwise we get a reactive client configured and then a health check that fails